src/app/modules/main/chat_section/chat_content/input_area/view.nim

```diff
@@ -154,3 +159,21 @@ QtObject:
   QtProperty[QVariant] urlsModel:
     read = getUrlsModel
     notify = urlsModelChanged
+
+  proc sendingInProgressChanged(self: View) {.signal.}
+  proc getSendingInProgress*(self: View): bool {.slot.} =
+    return self.sendingInProgress
+
+  QtProperty[bool] sendingInProgress:
+    read = getSendingInProgress
+    notify = sendingInProgressChanged
+
+  proc setSendingInProgress*(self: View, value: bool) =
```

Comment:
I am wondering what will happen if I send big set of images twice. Will the first response(SIGNAL_SENDING_SUCCESS) set sendingInProgress to false while the second is still in progress?